Automattic/wp-calypso
# Licensing portal: Rename the existing payment methods state

#### Changes proposed in this Pull Request

Rename the existing `payment-methods` state to match the actual behavior. In this code, we are not describing the payment methods but the logic around adding a new credit card form.

#### Testing instructions

- If you do not have a partner key, please get in touch with Avalon for one, or you will be unable to view the UI.
- Checkout PR locally, run yarn && yarn start-jetpack-cloud.
- Visit http://jetpack.cloud.localhost:3000/partner-portal/payment-methods and select your partner key, if prompted.
- Click the “Add new credit card” button.
- Verify that the form is displayed and no errors are shown in the console
